### DISCUSSION TITLE
Unofficial effort to create a list of volunteers who could test Bevy on mobile hardware.

### DISCUSSION THREAD

**Discussion by inodentry** [bevyengine/bevy#4559] [General]
I have just created a new repository: https://github.com/IyesGames/bevy-community-mobile-testers/blob/main/README.md

The idea is to create a list of mobile hardware that active members of the Bevy community have available to test Bevy on. If Bevy developers need to test something on specific hardware, these might be people they could attempt to contact.

If you would like to be included, please create a PR to add yourself. Please only do it if you are actually willing to help, and keep your information updated.

**Comment by Nilirad** [comment]
I think that there should be a checkbox to be checked for those to be willing to run closed source code. Maybe someone could be hesitant to run a random application that cannot be audited.

**Reply by inodentry** [reply]
Just to clarify: the idea is *not* "ppl can contact you to run their bevy apps on your phone". That would be *bad*.

The idea was: "Bevy engine contributors can ask you to run some of the official bevy examples, or try a PR, ...", stuff like that. This would be of great help to make sure that bevy itself isn't broken in hardware/driver specific ways.

This is to help Bevy engine dev, which is open-source. Not to be a tester for random people's bevy projects.

**Reply by Nilirad** [reply]
Ah, ok, now it makes sense 👍🏻
